config/config.go

```diff
@@ -350,7 +357,23 @@ func IsTLSenabled() bool {
 	return false
 }
 
-func realKey(key string) string {
+func firstKeySet(keys ...string) string {
+	if len(keys) == 0 {
+		fmt.Fprintf(os.Stderr, "Was expecting at least one key")
+		return ""
+	}
+	var k string
+	for _, k = range keys {
+		if cfg.IsSet(k) {
+			return k
+		}
+	}
+	return k
```

Comment:
But this will change the current behavior which does not report attempting to read non-defined keys ... for example currently if run `GetInt("a")` no warning will be emitted. 

If you only want `GetInt("a", "b")` to emit a warming but not `GetInt("a")` then this would make  things inconsistent.

If you want both `GetInt("a", "b")` and `GetInt("a")` to emit warnings if keys not set - they this is a change that should be done outside the scope of this PR (as it will require changing at least one non-related test in config_test.go

Reply:
Ok , then let's remove the warning (the current behavior) and return ""

Reply:
ok, so I am returning "" (other than that leaving "as-is")